core/src/main/scala/pureconfig/package.scala

```diff
@@ -154,4 +261,14 @@ package object pureconfig {
       loadConfig[Config](resolvedTypesafeConfig)
     }
   }
+
+  /**
+   * @see [[loadConfigFromFiles]]
```

Comment:
Maybe we should use the annotation see to avoid duplicating the documentation everywhere because every load method has two versions. Duplicating the documentation is not useful.